capture_service/dive_client_cli.cc

```diff
@@ -710,7 +703,7 @@ absl::Status CmdPm4Capture(const CommandContext& ctx)
     }
 
     std::cout << "Waiting " << ctx.options.trigger_capture_after << " seconds...\n";
-    std::this_thread::sleep_for(std::chrono::seconds(ctx.options.trigger_capture_after));
+    absl::SleepFor(absl::Seconds(ctx.options.trigger_capture_after));
```

Comment:
Don't touch PM4 case for code that cleans up GFXR capture. Do this in a separate PR
```suggestion
    std::this_thread::sleep_for(std::chrono::seconds(ctx.options.trigger_capture_after));
```